src/Hyperwallet/Hyperwallet.php

```diff
@@ -2398,4 +2396,27 @@ public function listBusinessStakeholders($userToken , $options) {
             return new BusinessStakeholder($entry);
         });
     }
+
+    //--------------------------------------
+    // Business Stakeholders
```

Comment:
was this moved here by mistake? seems like everything above is business stakeholders and below is transfer methods

Reply:
thanks, i changed the comments